src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
```

Comment:
Don't use static methods or static variables in your solution. Instead, you should create an instance of `Random` in the method where it's used.